### PR TITLE
Adding missing parameters to docs and inspect components, sleep param

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Missing documentation for a couple of options in cerberus 

Also added option for inspect components

### CODE REVIEW

File: docs/cerberus.md [hunk 1]
```diff
@@ -20,10 +20,13 @@ The following environment variables can be set on the host running the container
 
 Parameter               | Description                                                           | Default
 ----------------------- | -----------------------------------------------------------------     | ------------------------------------ |
-KUBECONFIG              | Path to the kubeconfig to access the cluster API                      | /root/.kube/config                   |
```
Comment (paigerube14) [inline docs/cerberus.md:23]: @chaitanyaenr do we want to actually add this into the template?

**Comment by paigerube14** [comment]
Added CERBERUS_KUBECONFIG variable, @chaitanyaenr please let me know your thoughts on this

File: docs/cerberus.md (hunk 1 above)
Comment (paigerube14) [reply docs/cerberus.md:23]: Talked with Ravi and don't want to expose this option in the documentation

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Review by yogananth-subramanian — COMMENTED** [review]
Thanks for the patch, was eyeing similar changes for running cerberus in k8s.

**Pull request merged**